src/appState.ts

```diff
@@ -17,6 +17,7 @@ export function getDefaultAppState(): AppState {
     elementType: "selection",
     elementLocked: false,
     exportBackground: true,
+    addWatermark: false,
```

Comment:
`add` implies an action. This is a state. Should be named something like `watermarkAdded` or something similar.

Reply:
includeWatermark?

Reply:
Good suggestion, I named it similar to `exportBackground` above it which I guess suffers the same problem. I settled on `shouldAddWatermark` since it reads nicely where it's used, though I'm happy to change it again.